x/go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/hibiken/asynq/x
 go 1.16
 
 require (
-	github.com/go-redis/redis/v8 v8.11.4
+	github.com/go-redis/redis/v8 v8.11.4 // indirect
```

Comment:
v8 is still being indirectly referenced in the x folder.

Reply:
Please fix

Reply:
@l0nax please fix.

Check module used with cmd: `go mod why github.com/go-redis/redis/v8`